controllers/articles.controllers.js

```diff
@@ -11,4 +11,12 @@ exports.getAllArticles = (req, res, next) => {
     getAllArticlesData().then((articles) => {
         res.status(200).send({ articles })
     })
+}
+
+exports.patchArticleById = (req, res, next) => {
+    const article_id = req.params.article_id;
```

Comment:
You could destructure here as you're calling the variable the same name as the object key has